### PR TITLE
[ES] Update `osu!supporter`

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/osu!supporter/es.md [hunk 1]
```diff
@@ -132,16 +139,16 @@ Los supporters pueden acceder al multijugador desde la versión de acceso antici
 
 La duración restante del osu!supporter, así mismo como la cantidad total de contribuciones, compras del mismo, y regalos, están disponibles en la parte superior de la [página de osu!supporter](https://osu.ppy.sh/home/support).
 
-## Conseguir osu!supporter
+## Conseguir un osu!supporter
```
Comment (agatemosu) [inline wiki/osu!supporter/es.md:142]:
```suggestion
## Conseguir osu!supporter
```

File: wiki/osu!supporter/es.md [hunk 2]
```diff
@@ -132,16 +139,16 @@ Los supporters pueden acceder al multijugador desde la versión de acceso antici
...
+Para conseguir un osu!supporter, visita la [tienda de osu!supporter](https://osu.ppy.sh/store/products/supporter-tag) y ajusta la duración usando el control deslizante o con los botones inferiores con el número de meses. Todos los precios están en dólares estadounidenses (USD) y no incluyen posibles cargos del sistema de pago.
 
-Después, haz clic en `Añadir al carrito` para agregar tu etiqueta osu!supporter al carrito de la osu!store. Para finalizar la compra, ve al [carrito](https://osu.ppy.sh/store/cart) y haz clic en `Pagar`, luego sigue las indicaciones de la pantalla.
+Después, haz clic en `Añadir al carrito` para agregar tu etiqueta de osu!supporter al carrito de la osu!store. Para finalizar la compra, ve al [carrito](https://osu.ppy.sh/store/cart) y haz clic en `Pagar`, luego sigue las indicaciones de la pantalla.
 
-### Regalar osu!supporter
+### Regalar un osu!supporter
```
Comment (agatemosu) [inline wiki/osu!supporter/es.md:148]:
```suggestion
### Regalar osu!supporter
```

File: wiki/osu!supporter/es.md [hunk 3]
```diff
@@ -132,16 +139,16 @@ Los supporters pueden acceder al multijugador desde la versión de acceso antici
...
 
 ::: Infobox
-![](img/profile-gift-supporter-ES.png "Botón con menú desplegable en una página de perfil para regalar a alguien un osu!supporter")
+![](img/profile-gift-supporter-ES.png "Botón con menú desplegable en una página de perfil para regalarle a alguien un osu!supporter")
```
Comment (agatemosu) [inline wiki/osu!supporter/es.md:151]:
```suggestion
![](img/profile-gift-supporter-ES.png "Botón con menú desplegable en una página de perfil para regalarle osu!supporter a alguien")
```

**Review by agatemosu — COMMENTED** [review]
1

File: wiki/osu!supporter/es.md [hunk 4]
```diff
@@ -41,30 +39,39 @@ Los supporters reciben una insignia con corazones en ella, usualmente se encuent
...
+- Banners, collabs, e infografías para darle vida al perfil
 - Algunos párrafos o artículos que proporcionen información sobre la vida del jugador
 - Un catálogo de mapas favoritos, mappers o cualquier otra cosa que consideren importante
 
 ### Portada de perfil
 
 Los supporters pueden personalizar aún más su perfil colocando una imagen de portada:
 
-- Haz clic en el icono del lápiz situado en la esquina inferior derecha de la portada.
+- Haz clic en el icono del lápiz en la esquina inferior derecha de la portada del perfil.
```
Comment (agatemosu) [inline wiki/osu!supporter/es.md:58]:
```suggestion
- Haz clic en el icono del lápiz situado en la esquina inferior derecha de la portada del perfil.
```

File: wiki/osu!supporter/es.md [hunk 5]
```diff
@@ -13,7 +11,7 @@ tags:
 
 ![Imagen promocional de osu!supporter](img/supporter-eyecatch-ES.jpg)
 
-**osu!supporter** (o la ***etiqueta osu!supporter***) es un título temporal dado a los jugadores que apoyan a osu!, de manera voluntaria, con una donación monetaria para ayudarlo a crecer. Los supporters reciben un puñado de beneficios cosméticos y técnicos, la mayoría solo se presentan cuando el título está presente. Las características extras de osu!supporter no dan ningún tipo de ventaja en el juego, rendimiento o puntuación, y nunca lo hará.
+**osu!supporter** (o la ***etiqueta osu!supporter***) es un título temporal dado a los jugadores que apoyan a osu! de manera voluntaria con una donación monetaria para ayudarlo a crecer. Los supporters reciben un puñado de beneficios cosméticos y técnicos, la mayoría solo se presentan cuando el título está presente. Las características extras de osu!supporter no dan ningún tipo de ventaja en el juego, rendimiento o puntuación, y nunca lo hará.
```
Comment (agatemosu) [inline wiki/osu!supporter/es.md:14]:
```suggestion
**osu!supporter** (o la ***etiqueta de osu!supporter***) es un título temporal dado a los jugadores que apoyan a osu! de manera voluntaria con una donación monetaria para ayudarlo a crecer. Los supporters reciben un puñado de beneficios cosméticos y técnicos, la mayoría solo se presentan cuando el título está presente. Las características extras de osu!supporter no dan ningún tipo de ventaja en el juego, rendimiento o puntuación, y nunca lo hará.
```

File: wiki/osu!supporter/es.md [hunk 6]
```diff
@@ -13,7 +11,7 @@ tags:
...
 
 La etiqueta osu!supporter podrá ser obtenida desde la [página de la tienda de osu!supporter](https://osu.ppy.sh/store/products/supporter-tag).
```
Comment (agatemosu) [inline wiki/osu!supporter/es.md:16]:
```suggestion
La etiqueta de osu!supporter podrá ser obtenida desde la [página de la tienda de osu!supporter](https://osu.ppy.sh/store/products/supporter-tag).
```

File: wiki/osu!supporter/es.md [hunk 7]
```diff
@@ -13,9 +11,9 @@ tags:
...
 
-**osu!supporter** (o la ***etiqueta osu!supporter***) es un título temporal dado a los jugadores que apoyan a osu!, de manera voluntaria, con una donación monetaria para ayudarlo a crecer. Los supporters reciben un puñado de beneficios cosméticos y técnicos, la mayoría solo se presentan cuando el título está presente. Las características extras de osu!supporter no dan ningún tipo de ventaja en el juego, rendimiento o puntuación, y nunca lo hará.
+**osu!supporter** (o la ***etiqueta de osu!supporter***) es un título temporal dado a los jugadores que apoyan a osu! de manera voluntaria con una donación monetaria para ayudarlo a crecer. Los supporters reciben un puñado de beneficios cosméticos y técnicos, la mayoría solo se presentan cuando el título está presente. Las características extras de osu!supporter no dan ningún tipo de ventaja en el juego, rendimiento o puntuación, y nunca lo hará.
 
-La etiqueta osu!supporter podrá ser obtenida desde la [página de la tienda de osu!supporter](https://osu.ppy.sh/store/products/supporter-tag).
+La etiqueta de osu!supporter podrá ser obtenida desde la [página de la tienda de osu!supporter](https://osu.ppy.sh/store/products/supporter-tag).
```
Comment (Unnombredeusuarioextremadamentelargo) [inline wiki/osu!supporter/es.md:16]:
```suggestion
La etiqueta de osu!supporter puede obtenerse en la [página de la tienda de osu!](https://osu.ppy.sh/store/products/supporter-tag).
```

File: wiki/osu!supporter/es.md [hunk 8]
```diff
@@ -41,30 +39,39 @@ Los supporters reciben una insignia con corazones en ella, usualmente se encuent
...
 - Selecciona una nueva portada a partir de unas cuantas opciones predefinidas o sube una imagen personalizada (sus dimensiones deberán ser de 2400x640).
 
-Una vez que el osu!supporter expire, la portada se mantendrá, pero no podrá cambiarse.
+Una vez que la etiqueta de osu!supporter expire, la portada se mantendrá, pero no podrá cambiarse.
+
+### Color del perfil
+
+Al igual que las portadas de los perfiles, los supporters pueden modificar el color de su perfil:
+
+- Haz clic en el icono del lápiz en la esquina inferior derecha de la portada del perfil.
```
Comment (Unnombredeusuarioextremadamentelargo) [inline wiki/osu!supporter/es.md:67]:
```suggestion
- Haz clic en el icono del lápiz situado en la esquina inferior derecha de la portada del perfil.
```

File: wiki/osu!supporter/es.md [hunk 9]
```diff
@@ -134,14 +141,14 @@ La duración restante del osu!supporter, así mismo como la cantidad total de co
...
 ## Conseguir osu!supporter
 
-Para conseguir un osu!supporter, visita la [tienda de osu!supporter](https://osu.ppy.sh/store/products/supporter-tag) y ajusta la duración utilizando el control deslizante o con los botones inferiores con el número de meses. Todos los precios están en dólares estadounidenses (USD) y no incluyen posibles cargos del sistema de pago.
+Para conseguir un osu!supporter, visita la [tienda de osu!supporter](https://osu.ppy.sh/store/products/supporter-tag) y ajusta la duración usando el control deslizante o con los botones inferiores con el número de meses. Todos los precios están en dólares estadounidenses (USD) y no incluyen posibles cargos del sistema de pago.
 
-Después, haz clic en `Añadir al carrito` para agregar tu etiqueta osu!supporter al carrito de la osu!store. Para finalizar la compra, ve al [carrito](https://osu.ppy.sh/store/cart) y haz clic en `Pagar`, luego sigue las indicaciones de la pantalla.
+Después, haz clic en `Añadir al carrito` para agregar tu etiqueta de osu!supporter al carrito de la osu!store. Para finalizar la compra, ve al [carrito](https://osu.ppy.sh/store/cart) y haz clic en `Pagar`, luego sigue las indicaciones de la pantalla.
```
Comment (Unnombredeusuarioextremadamentelargo) [inline wiki/osu!supporter/es.md:146]:
```suggestion
Después, haz clic en `Añadir al carrito` para añadir tu etiqueta de osu!supporter al carrito de la osu!store. Para finalizar la compra, ve al [carrito](https://osu.ppy.sh/store/cart) y haz clic en `Pagar`, luego sigue las indicaciones de la pantalla.
```

File: wiki/osu!supporter/es.md [hunk 10]
```diff
@@ -134,14 +141,14 @@ La duración restante del osu!supporter, así mismo como la cantidad total de co
...
 
 ### Regalar osu!supporter
 
 ::: Infobox
-![](img/profile-gift-supporter-ES.png "Botón con menú desplegable en una página de perfil para regalar a alguien un osu!supporter")
+![](img/profile-gift-supporter-ES.png "Botón con menú desplegable en una página de perfil para regalarle osu!supporter a alguien")
```
Comment (Unnombredeusuarioextremadamentelargo) [inline wiki/osu!supporter/es.md:151]:
```suggestion
![](img/profile-gift-supporter-ES.png "Botón con menú desplegable en una página de perfil para regalarle un osu!supporter a alguien")
```

**Review by Unnombredeusuarioextremadamentelargo — CHANGES_REQUESTED** [review]

**Review by agatemosu — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**